File: tools/bench/answers/go_sonnet/s3/06_stats.go
package main

import "fmt"

func main() {
	nums := []float64{4, 8, 15, 16, 23}
	min, max, sum := nums[0], nums[0], 0.0
	for _, n := range nums {
		if n < min {
			min = n
		}
		if n > max {
			max = n
		}
		sum += n
	}
	fmt.Println(min)
	fmt.Println(max)
	fmt.Println(sum / float64(len(nums)))
}

File: tools/bench/answers/rust_sonnet/s1/06_stats.rs
fn main() {
    let nums = [4, 8, 15, 16, 23];
    let min = nums.iter().min().unwrap();
    let max = nums.iter().max().unwrap();
    let sum: i64 = nums.iter().sum();
    let mean = sum as f64 / nums.len() as f64;
    println!("{}", min);
    println!("{}", max);
    println!("{}", mean);
}
